os-appium-tests/tests/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_TIMEOUT = 10000;
+export const DEFAULT_TIMEOUT_INTERVAL = 1000;
```

Comment:
Can you explain the difference between DEFAULT_TIMEOUT and DEFAULT_TIMEOUT_INTERVAL? Can we have a better name?

Reply:
@pmaroco  These two values came from the framework - we only make it work adding the export ...  :(, it's no excuse but the comment imo should be changed or improved at the framework